### DISCUSSION TITLE
ktor STOMP server

### DISCUSSION THREAD

**Discussion by chriswu000** [joffrey-bion/krossbow#104] [General]
Hi!

Apologies if this is a bad question,  but I am new to websockets, STOMP, and ktor, so my understanding is still limited.  Can I use krossbow with ktor to build a STOMP server, or is it meant specifically for clients?  Thanks!

**Comment by joffrey-bion** [comment]
Hi, thanks for reaching out, this is definitely a good question, no need to apologize at all!

This specific library is only a client library.
The krossbow-websocket-ktor artifact is to make the Krossbow STOMP client use Ktor's web socket client implementation as transport.

I don't believe there is an implementation of STOMP for Ktor server out there at the moment, unfortunately.

**Reply by chriswu000** [reply]
Hi, thanks for the info, appreciate it!

**Comment by joffrey-bion** [comment]
FYI, there is a feature request for this in KTOR issues:
https://youtrack.jetbrains.com/issue/KTOR-6876
